src/services/CleanupService.ts

```diff
@@ -46,7 +48,7 @@ export class CleanupService {
         if (moment(new Date(visit.startTime)).isAfter(recentActivityWindowTime)) {
           // Visit open less than 3 hours, no action needed
           this.addToLogVisits(
-            visit.id,
+            visit.id!,
```

Comment:
why is the ! required here, won't there always be a visit id?

Reply:
We know at this point it will always be populated, as it has come from the table, however id is optional in the new schema.

Reply:
I don't think that the schema is correct in that case, FAIK the activity id is mandatory after they've been created as it gets automatically generated

Reply:
Looks like we use the activity schema before the ID is generated, meaning it would have to be optional?

Reply:
Yes exactly, that was my understanding too.

Reply:
there might be an argument for using something else until its generated, then changing the schema so its required (assuming its always required once generated).  Theres quite a lot of validation before its generated though.